src/modules/cursor.c

```diff
@@ -105,6 +105,31 @@ static void printCursorXFCE(FFinstance* instance)
     ffStrbufDestroy(&cursorSize);
 }
 
+static void printCursorLXQt(FFinstance* instance)
+{
+    FFstrbuf cursorTheme;
+    ffStrbufInit(&cursorTheme);
+
+    FFstrbuf cursorSize;
+    ffStrbufInit(&cursorSize);
+
+    if(ffParsePropFileConfigValues(instance, "/lxqt/session.conf", 2, (FFpropquery[]) {
+        {"cursor_theme =", &cursorTheme},
+        {"cursor_size =", &cursorSize}
+    })) {
+    }
+
+    if(cursorTheme.length == 0)
+    {
+        ffPrintError(instance, FF_CURSOR_MODULE_NAME, 0, &instance->config.cursorKey, &instance->config.cursorFormat, FF_CURSOR_NUM_FORMAT_ARGS, "Couldn't find LXQt cursor in session.conf");
+        return;
```

Comment:
`cursorTheme` and `cursorSize` must be destroyed here. I admit i forgot that in the plasma code, can you add that there  too please?